SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security policy
+
+## Supported versions
+
+Security updates will be released after vulnerabilities have been identified and fixed. At this point in time,
+we will only release security patches for charm libraries to their latest major versions released on Charmhub.
```

Comment:
Similar to the `slurmutils` _SECURITY.md_, support latest two major versions, but scoped to only the charm libraries released to Charmhub:

```suggestion
Security updates for charm libraries will be released for the latest two major versions released on Charmhub.
```

Note that this might change in the future depending CharmTechs work on Charm SDKs to get rid of complex charm libraries that rely on PYDEPS.